### PR TITLE
Added test for mandatory Organization selection in order to view Acce…

### PR DESCRIPTION
**Pull request by anarang**
…ss Insights.

Edited 'test-docstrings' in 'Makefile' to add tests/foreman/rhai.

### CODE REVIEW

File: robottelo/ui/rhai.py [hunk 1]
```diff
@@ -12,3 +12,13 @@ def view_registered_systems(self):
...
         return result
+
+    def org_selection_required(self):
+        """Message: 'Organization Selection Required' must be displayed if
+        the user tries to view Access Insights overview without selecting an
+        org"""
+
+        result = self.wait_until_element(
+            locators['insights.org_selection_msg']
+        ).text
```
Comment (sthirugn) [inline robottelo/ui/rhai.py]: I am not sure if this function is good enough to be a separate utility method.  Can't this line be done in the test itself?

**Comment by sthirugn** [comment]
ACK pending comment.

File: robottelo/ui/rhai.py (hunk 1 above)
Comment (anarang) [inline robottelo/ui/rhai.py]: Yes, will make the changes.

File: tests/foreman/rhai/test_rhai.py [hunk 2]
```diff
@@ -98,3 +99,25 @@ def test_rhai_client_setup(self):
...
+
+        with Session(self.browser) as session:
+            # Given that the user does not specify any Organization
+            session.nav.go_to_select_org("Any Organization")
+            session.nav.go_to_insights_overview()
+
+            # 'Organization Selection Required' message must be present
+            result = session.nav.wait_until_element(
+                locators['insights.org_selection_msg']).text
+            self.assertIn("Organization Selection Required", result)
```
Comment (sthirugn) [inline tests/foreman/rhai/test_rhai.py]: This test will fail if you run in any language other than english.  Is it safe to assume that you test only english?

Comment (anarang) [inline tests/foreman/rhai/test_rhai.py]: Yes, will be run only in English for now. I shall take care of the language part sometime later, not an immediate concern though.

**Comment by sthirugn** [comment]
ACK

File: tests/foreman/rhai/test_rhai.py [hunk 3]
```diff
@@ -98,3 +99,25 @@ def test_rhai_client_setup(self):
...
+        select an Organization if there is no organization selected
+
+        @Feature: In order to use Access Insights, user must select an
+        organization
+
+        @Assert: 'Organization Selection Required' message must be displayed if
+        the user tries to view Access Insights overview without selecting an
+        org
+        """
+
```
Comment (Ichimonji10) [inline tests/foreman/rhai/test_rhai.py]: Nitpick: move the triple quotes down a line, like so:

    @Assert: 'Organization Selection Required' message must be displayed if
    the user tries to view Access Insights overview without selecting an
    org

    """
    with Session(self.browser) as session:

**Comment by Ichimonji10** [comment]
ACK pending nitpick.

File: tests/foreman/rhai/test_rhai.py (hunk 2 above)
Comment (oshtaier) [inline tests/foreman/rhai/test_rhai.py]: Basically, it is convention for us to omit any connection to specific language messages, so if you run your tests on Jenkins, statement 'will be run only in English for now' is pretty incorrect (or can be in future).

Comment (sthirugn) [inline tests/foreman/rhai/test_rhai.py]: @oshtaier right now, rhai tests are not run with regular robottelo tests.  A new jenkins job will be created soon which runs just robottelo/tests/foreman/rhai/ tests.

Comment (anarang) [inline tests/foreman/rhai/test_rhai.py]: @oshtaier by 'right now' I meant the same as @sthirugn mentioned.

**Comment by anarang** [comment]
@Ichimonji10 nitpick update done. :)

**Comment by kbidarkar** [comment]
I see 2 ACK's and nitpick also updated as per @anarang comment.
so Merging.

**Pull request merged**